API/templates/home.html

```diff
@@ -14,11 +14,13 @@
        <tr>
         <th>Sensor Name</th>
         <th>Sensor Unit</th>
+        <th>Delete</th>
        </tr>
        {% for sensor in sensors %}
        <tr>
         <td>{{sensor[2]}}</td>
         <td>{{sensor[3]}}</td>
+        <td><a href="/delete/{{sensor[0]}}" class="btn btn-outline-danger btn-sm">Delete</a></a> </td>
```

Comment:
Why do we have a class "btn btn-outline-danger..." ? 
Why there are two closing </a> tags ?